runtime-light/allocator/runtime-light-allocator.cpp

```diff
@@ -61,65 +63,56 @@ void RuntimeAllocator::free() {
 }
 
 void* RuntimeAllocator::alloc_script_memory(size_t size) noexcept {
-  php_assert(size > 0);
```

Comment:
are you sure that's the right thing to do? Seems like a logical check

Reply:
Generally it should not happen, but let's add them back